test/integration/.import-restrictions

```diff
@@ -0,0 +1,8 @@
+{
+	"Rules": [
+		{
+			"SelectorRegexp": "k8s[.]io/kubernetes/test/e2e",
+			"AllowedPrefixes": []

```

Comment:
This means `test/integration` code is not allowed to import any code under `test/e2e/`.